### PR TITLE
[Style/#33]:카탈로그 목록 조회 페이지 구현

### PR DESCRIPTION
**Pull request by juhee067**
- 뱃지 카테고리 생성
- filter ui 구현
- 술 정보 카드 생성 : 카드 위에 붙는 header에 따라 카드 형태 다름
- 검색창
- CatalogPage NavContentLayout 분리

<br/>

### ✔️ 뱃지 카테고리 컴포넌트 => 수정 필요 (API)
image

```js
    <Category type='catalog' />
```

### ✔️ 술 정보 카드 컴포넌트
image

```js
 <List>
        {liguorData.map((item) => {
          return <ProductCard key={item.title} liquorInfo={item} />;
        })}
      </List>
```

### ✔️ 검색창 컴포넌트 
image

## 📌 구현 결과 (선택)

image

<br/>

## 📌 기타 사항

- 메인에 사용했던 뱃지 카테고리와 카드 부분 다시 가지고 와서 사용하시면 될 것 같습니다! 세종님이 만드셨던 부분 가져와서 수정했습니다.
- 고정돼서 내려오도록 수정 (WIP)
image

<br/>

### CODE REVIEW

File: src/components/Header/Category.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+interface CategoryProps {
+  type: CategoryTypeKey;
+}
+
+function Category({ type }: CategoryProps) {
+  const category = Object.values(CATEGORY[type]);
+  const [selectedCategory, setSelectedCategory] = useState<string>(category[0]);
+  const handleClickButton = (categroy: string) => {
+    setSelectedCategory(categroy);
```
Comment (JIMIN1020) [inline src/components/Header/Category.tsx:17]: 검색, 카테고리 필터링 같은 건 url query string으로 관리하면 좋을 것 같아요!

File: src/features/Catalog/components/Filter.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import styled, { useTheme } from 'styled-components';
+import { DownArrowIcon } from 'public/assets/icons';
+import { FILTER } from '../constants/filter';
+
+function Filter() {
+  const [selected, setSelected] = useState<{ [key: string]: boolean }>({
+    alcohol: false,
+    price: false,
+    rating: false,
+  });
```
Comment (JIMIN1020) [inline src/features/Catalog/components/Filter.tsx:11]: 여기도 마찬가지로 query string으로 관리하는 게 좋을 것 같습니다!

File: src/pages/CatalogPage.tsx [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+          <ProductGrid />
+        </div>
+      </Content>
+    </Container>
+  );
 }
 
 export default CatalogPage;
+
+const Container = styled.div``;
```
Comment (JIMIN1020) [inline src/pages/CatalogPage.tsx:22]: 여기서 Container는 스타일링이 없는데 그냥 div나 fragment로 바꿔도 되지 않나요?!

**Review by JIMIN1020 — APPROVED** [review]
고생 많으셨습니다!!
코드가 보기 편하네요 ㅎㅎ

File: src/pages/CatalogPage.tsx (hunk 3 above)
Comment (juhee067) [reply src/pages/CatalogPage.tsx:22]: 넵 알겠습니다!

File: src/components/Header/Category.tsx (hunk 1 above)
Comment (juhee067) [reply src/components/Header/Category.tsx:17]: 아아 네네!! 알겠습니다!

File: src/features/Catalog/components/Main/ProductGrid.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        })}
+      </List>
+      {/* TODO:페이지네이션 추가 */}
+    </Container>
+  );
+}
+
+export default ProductGrid;
+
+const Container = styled.div``;
```
Comment (SebellKo) [inline src/features/Catalog/components/Main/ProductGrid.tsx:48]: 요부분 프라그먼트로 대체하는게 어떨까요 ?!

추후 스타일 추가가 있을 예정일까요 ?!?!

File: src/pages/CatalogPage.tsx (hunk 3 above)
Comment (SebellKo) [reply src/pages/CatalogPage.tsx:22]: 아차차 여기 있었네요 ㅎㅎ

**Review by SebellKo — APPROVED** [review]
주희님 코드 확인했습니다 ~

너무 고생 많으셨습니다 😇😇

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">0%</td> <td align="right">0 / 2886</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">0%</td> <td align="right">0 / 2886</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">0.86%</td> <td align="right">1 / 115</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">0.86%</td> <td align="right">1 / 115</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx">src/components/Header/Category.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L4-L5" class="text-red">4-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L13-L18" class="text-red">13-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L20-L29" class="text-red">20-29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L31-L34" class="text-red">31-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L36" class="text-red">36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Category.tsx#L40" class="text-red">40</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Search.tsx">src/components/Header/Search.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Search.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Search.tsx#L4-L8" class="text-red">4-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Search.tsx#L10" class="text-red">10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Header/Search.tsx#L12-L13" class="text-red">12-13</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx">src/components/Input/FormInput.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L10-L29" class="text-red">10-29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L31-L32" class="text-red">31-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L34" class="text-red">34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L36-L37" class="text-red">36-37</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L39" class="text-red">39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L41" class="text-red">41</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L46" class="text-red">46</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L48" class="text-red">48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L50-L51" class="text-red">50-51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L58" class="text-red">58</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L62" class="text-red">62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/components/Input/FormInput.tsx#L69" class="text-red">69</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/constants/category.ts">src/constants/category.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/constants/category.ts#L1-L27" class="text-red">1-27</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/index.ts">src/features/Catalog/index.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/index.ts#L1" class="text-red">1</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx">src/features/Catalog/components/Filter.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L6-L12" class="text-red">6-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L14-L16" class="text-red">14-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L18-L27" class="text-red">18-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L29-L39" class="text-red">29-39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L41-L43" class="text-red">41-43</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L45" class="text-red">45</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L47" class="text-red">47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L49" class="text-red">49</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L51" class="text-red">51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L53-L55" class="text-red">53-55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L58-L59" class="text-red">58-59</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L62" class="text-red">62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L65-L66" class="text-red">65-66</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L70-L71" class="text-red">70-71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L74" class="text-red">74</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Filter.tsx#L76-L77" class="text-red">76-77</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx">src/features/Catalog/components/Header/CatalogHeader.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L5-L13" class="text-red">5-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L15" class="text-red">15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L17" class="text-red">17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L19" class="text-red">19</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L22" class="text-red">22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Header/CatalogHeader.tsx#L29-L31" class="text-red">29-31</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx">src/features/Catalog/components/Main/ProductCard.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L17-L29" class="text-red">17-29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L31" class="text-red">31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L33-L35" class="text-red">33-35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L50" class="text-red">50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L58-L60" class="text-red">58-60</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductCard.tsx#L63" class="text-red">63</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx">src/features/Catalog/components/Main/ProductGrid.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L7-L32" class="text-red">7-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L34-L40" class="text-red">34-40</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L42" class="text-red">42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L44" class="text-red">44</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L46" class="text-red">46</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/components/Main/ProductGrid.tsx#L48-L49" class="text-red">48-49</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/constants/filter.ts">src/features/Catalog/constants/filter.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/constants/filter.ts#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Catalog/constants/filter.ts#L6-L25" class="text-red">6-25</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts">src/features/Join/api/signup.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L7-L9" class="text-red">7-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L11-L14" class="text-red">11-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L16-L18" class="text-red">16-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L20-L23" class="text-red">20-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/features/Join/api/signup.api.ts#L25-L34" class="text-red">25-34</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx">src/layouts/NavContentLayout.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx#L6-L14" class="text-red">6-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx#L16" class="text-red">16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx#L18" class="text-red">18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/layouts/NavContentLayout.tsx#L20" class="text-red">20</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx">src/pages/CatalogPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx#L5-L16" class="text-red">5-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx#L18" class="text-red">18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx#L20" class="text-red">20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/pages/CatalogPage.tsx#L22-L23" class="text-red">22-23</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/routes/index.tsx">src/routes/index.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/routes/index.tsx#L1-L13" class="text-red">1-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/routes/index.tsx#L15-L67" class="text-red">15-67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/routes/index.tsx#L69-L71" class="text-red">69-71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8f063f8b59a119da0a3e51c70a3e89abc9cc6520/src/routes/index.tsx#L73" class="text-red">73</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10246727420>#53</a></em>

**Pull request merged**